src/style.css

```diff
@@ -0,0 +1,169 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
+
+* {
```

Comment:
- [ ] [OPTIONAL] You can duplicate the provided wireframe as much as possible, kindly center your elements on the page and align the submit button to the right of the form, so your design looks more professional
Note: Your design is great 💯 just try to make use of the white space so the page will not look cluttered